presentation/chapters/de-DE/documentation.chapter

```diff
@@ -0,0 +1,95 @@
+# Dokumentation
+
+---
+
+## `rustdoc`
+
+Rust bietet ein Standarddokumentationswerkzeug namens 'rustdoc`. Es wird häufig durch `cargo doc` verwendet.
+
+Aufgrund dieser Rust-Code ist fast immer in einem gemeinsamen Format dokumentiert.
+
+---
+
+## `std` Dokumentation
+
+Die Standard-Bibliotheksdokumentation wird unter https://doc.rust-lang.org/std/ gehostet.
+
+Eine lokale Offline-Version kann geöffnet werden mit:
+
+```bash
+rustup doc --std
+```
+
+---
+
+## Crate Dokumentation
+
+Dokumentation für Crates unter http://crates.io/ finden Sie unter https://docs.rs/.
+
+Einige Crates können auch andere Dokumente über den Link "Dokumentation" auf ihrer Liste in http://crates.io/ gefunden haben.
+
+---
+
+## Beispiel: Ein Modul
+
+<iframe src="https://doc.rust-lang.org/std/vec/" width=100% height=400></iframe>
+
+---
+
+## Beispiel: Ein Modul
+
+Diese Seite dokumentiert das `vec` Modul.
+
+Es beginnt mit einigen Beispielen, dann listet alle `struct`s, Eigenschaften oder Funktionen, die das Modul exportiert.
+
+---
+
+## Wie wird es generiert?
+
+`rustdoc` kann Rust-Code und Markdown-Dokumente lesen.
+
+`//!` Und `///` Kommentare werden als Markdown gelesen.
+
+```rust
+//! Moduldokumentationskommentare wie der Abschnitt 'Examples' von `std::vec`.
+
+/// Dokumente Funktionen, `struct`s und Eigenschaften.
+/// Hier dokumentiert es eine Funktion.
+fn function_with_documentation() {}
+
+// Zeigt auch in der Dokumentation, aber nicht mit einem Kommentar.
```

Comment:
I'm trying to explain that the comment with `//` won't show up in the docs, but the function will.